google/cloud/storage/internal/grpc_client.cc

```diff
@@ -205,11 +203,8 @@ StatusOr<ObjectMetadata> GrpcClient::InsertObjectMedia(
   google::storage::v1::Object response;
   auto stream = stub_->InsertObject(&context, &response);
   auto proto_request = ToProto(request);
-  // This limit is for the *message*, not just the payload. It includes any
-  // additional information such as checksums. We need to use a stricter limit,
-  // a chunk quantum seems to work in practice.
   std::size_t const maximum_buffer_size =
-      kMaxInsertObjectWriteRequestSize - UploadChunkRequest::kChunkSizeQuantum;
+      google::storage::v1::ServiceConstants_Values_MAX_WRITE_CHUNK_BYTES;
```

Comment:
I think we can avoid using the `*_Values_*` enums by accessing the named fields in the `ServiceConstants` class. Like this could be `google::storage::v1::ServiceConstants::MAX_WRITE_CHUNK_BYTES`.

Reply:
Thanks, fixed.